.travis.yml

```diff
@@ -4,8 +4,9 @@ matrix:
   include:
     - python: 3.5
       env: TOXENV=py35
+  allow_failures:
+  - python: 'pypy3'
```

Comment:
@pgrunewald seems that this is not the syntax travis expects, I guess, that you should put ``TOXENV=pypy3`` here